packages/console/src/pages/Profile/containers/BasicUserInfoUpdateModal/index.tsx

```diff
@@ -121,7 +125,15 @@ const BasicUserInfoUpdateModal = ({ field, value: initialValue, isOpen, onClose
         }
         onClose={onClose}
       >
-        <div>
+        {field === 'avatar' && isUserAssetsServiceReady ? (
```

Comment:
just isUserAssetsServiceReady ?

Reply:
No, `avatar`, `name`, and `username` share the same update modal.